main.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"log"
+	"net/http"
+	"os"
+)
+
+var (
+	airtableApiKey = os.Getenv("AIRTABLE_API_KEY")
+	airtableBaseId = os.Getenv("AIRTABLE_BASE_ID")
```

Comment:
Go naming-wise: initialisms must be all-caps.

    airtableAPIKey
    airtableBaseID

https://github.com/golang/go/wiki/CodeReviewComments#initialisms